reference/sectionC_interpretation.inc

```diff
@@ -42,83 +44,87 @@ Units reference
-| radian        | 1, 1              | (metre, 1), (metre, -1)        |
-+---------------+-------------------+--------------------------------+
-| second        | -                 | -                              |
-+---------------+-------------------+--------------------------------+
-| siemens       | 1, 1, 1, 1        | (metre, -2), (kilogram -1),    |
-|               |                   | (second, 3), (ampere, 2)       |
-+---------------+-------------------+--------------------------------+
-| sievert       | 1, 1              | (metre, 2), (second, -2)       |
-+---------------+-------------------+--------------------------------+
-| steradian     | 1, 1              | (metre, 2), (metre, -2)        |
-+---------------+-------------------+--------------------------------+
-| tesla         | 1, 1, 1           | (kilogram, 1), (second, -2),   |
-|               |                   | (ampere, -1)                   |
-+---------------+-------------------+--------------------------------+
-| volt          | 1, 1, 1, 1        | (metre, 2), (kilogram, 1),     |
-|               |                   | (second , -3), (ampere, -1)    |
-+---------------+-------------------+--------------------------------+
-| watt          | 1, 1, 1           | (metre, 2), (kilogram, 1),     |
-|               |                   | (second, -3)                   |
-+---------------+-------------------+--------------------------------+
-| weber         | 1, 1, 1, 1        | (metre, 2), (kilogram, 1),     |
-|               |                   | (second, -2), (ampere, -1)     |
-+---------------+-------------------+--------------------------------+
++---------------+-------------------+-------------------------------------------+------------------------------------+
+| **Name**      | **Symbol**        | | **Unit reduction tuple**                | **Common definitions**             |
+|               |                   | | multiplier⋅(base, exponent)             |                                    |
++---------------+-------------------+-------------------------------------------+------------------------------------+
+| ampere        | :math:`A`         | \-                                        |                                    |
++---------------+-------------------+-------------------------------------------+------------------------------------+
+| becquerel     | :math:`Bq`        | (second, -1)                              | :math:`1/s`                        |
++---------------+-------------------+-------------------------------------------+------------------------------------+
+| candela       | :math:`cd`        | \-                                        |                                    |
++---------------+-------------------+-------------------------------------------+------------------------------------+
+| coulomb       | :math:`C`         | (second, 1), (ampere,1)                   | | :math:`s⋅A`                      |
```

Comment:
I can see that, in commit 333b6f7, you replaced `\cdot` with `⋅` in `math` constructs. I don't think you should do that. It messes up the rendering. `⋅` should only be used in "normal" text, not within a `math` construct.

Reply:
I haven't seen the difference?  I was hoping that your magic dot would render smaller and with less space than the \cdot symbol, but as they're both coming out the same I gave up!  I've searched for a smaller latex symbol than \cdot, but can't seem to find anything that looks good ... if you have a suggestion am happy to fix it, but because these tables are *such* a massive timesink, I really don't want to just play around ...

Reply:
All I am saying is that we should use the `:math:`  construct the way it is intended to be used, i.e. using `\cdot` (or something else of the sort although I am not sure there is something better for what we want to do here) and let the renderer do its thing, and `⋅` for "normal" text because that's the best we can do in that case.

Reply:
Right, I've nuked it all.  Much cleaner now, I hope (see 9fa5100) If not, please make whatever changes you need to ... this table has shortened my life enough!

Reply:
> this table has shortened my life enough!

:)